packages/financial-templates-lib/src/logger/DiscordTransport.ts

```diff
@@ -25,11 +31,15 @@ export class DiscordTransport extends Transport {
         embeds: [{ title: `${info.at}: ${info.message}`, description: this.formatLinks(info.mrkdwn), color: 9696729 }],
       };
 
-      // Select webhook url based on escalation path. if escalation path is not found, use default webhook url.
-      const webHook = this.escalationPathWebhookUrls[info.notificationPath] ?? this.defaultWebHookUrl;
+      // Select webhook from escalation path. If no escalation, use default webhook url, if postOnNonEscalationPaths is
+      // true.Else, if postOnNonEscalationPaths is false and there is no escalation path, do not post. This enables the
+      // transport to be configured to only send messages that have a specific escalation path.
+      if (this.postOnNonEscalationPaths && this.escalationPathWebhookUrls[info.notificationPath] != undefined) {
```

Comment:
nit: did you intentionally put `!=` vs `!==` so that you check for null as well? If so, I think `this.postOnNonEscalationPaths && this.escalationPathWebhookUrls[info.notificationPath]` would be clearer (i.e. no != at all). If you only want to check for undefined, I would use `!==`.

Reply:
ye, good point. will revert to your syntax.